### ISSUE TITLE
Support require() without callback ("Cannot read property 'apply' of undefined" error)

### ISSUE THREAD

**Issue by pshendry** [OpenNTF/dojo-webpack-plugin#158]
My project uses ASP.NET views which load their Dojo page content by importing the component in a script tag:

```js
<script type="text/javascript">
  require(["abc/editor"]);
</script>
```

I'm pretty sure that using `require` in this way (without a callback function) is valid. Doing this under `dojo-webpack-plugin` however causes an error: `"Uncaught TypeError: Cannot read property 'apply' of undefined"`. The offending snippet is in [DojoAMDMainTemplate.runtime.js](https://github.com/OpenNTF/dojo-webpack-plugin/blob/1c59c3a932bfbaeb04ee0a671a71912110afe9f9/lib/DojoAMDMainTemplate.runtime.js#L135-L137); looks like it just needs to check that `callback` is defined before executing it.

In the meantime there's a natural workaround of passing an empty function:

```js
<script type="text/javascript">
  require(["abc/editor"], function() {});
</script>
```

Happy to put up a PR once I'm home and reunited with my SSH key.

**Comment by pshendry** [comment]
Thanks! Great project by the way!

**Comment by chuckdumont** [comment]
You're welcome, and thanks.  This is fixed in release 2.7.2
